packages/cojson/src/storage/knownState.ts

```diff
@@ -0,0 +1,70 @@
+import { getIsUploaded } from "../SyncStateManager.js";
+import { RawCoID } from "../ids.js";
+import { CoValueKnownState, emptyKnownState } from "../sync.js";
+
+export class StorageKnownState {
```

Comment:
Used to track how much data we have in Storage and to wait for sync with storage.